src/libraries/System.Runtime/tests/System/Type/TypePropertyTests.cs

```diff
@@ -195,6 +195,7 @@ public void IsArray_Get_ReturnsExpected()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34196", TestRuntimes.Mono)]
```

Comment:
I would have expected the problematic test to be already excluded by

https://github.com/dotnet/runtime/blob/b74422f2d528937830ae50b241aeb3d868979228/src/libraries/System.Runtime/tests/System/Type/TypePropertyTests.cs#L534

Is xharness handling the `ActiveIssue` attribute on per-type basis?

Reply:
Good point! Let me take a look, ActiveIssue support is a bit hacky as far as I understand with xharness

Reply:
Looks like the workaround for https://github.com/dotnet/xharness/issues/156 looks for the attribute only on the methods and not on the type.

Reply:
We should be able to get rid of that workaround now I think.